src/main/scala/Monday_Basic.scala

```diff
@@ -0,0 +1,150 @@
+object Monday_Basic extends App {
+
+  //Basic
+  //Output "Hello World via the println() method"
+  println("Hello World")
+
+  //Store "Hello World!" in a variable and then output it
+  var output = "Hello World"
+  println(output)
+
+  //Create a method that accepts a string as a parameter and then outputs that string
+  def output(word: String) = {
+    println(word)
+  }
+
+  output("William")
+
+  //Create a method that returns a string, in this case “Hello World!”, which you then use to output
+  def returnOutput(): String = {
+    var name = "Manchester"
+    return name
```

Comment:
we don't use keyword return in scala, last line or expression will be returned

the whole method body could be just "Manchester" and it would do the same thing